apps/koa-server/project.json

```
@@ -4,7 +4,7 @@
   "targets": {
     "watch": {
       "options": {
-        "commands": ["tsc --watch", "nodemon dist/server.js"]
+        "commands": ["bunx tsc --watch", "bunx nodemon dist/server.js"]
```

Read comment below i think you could change this to the following

```suggestion
        "commands": ["bun run --watch server.ts"]
```

---

Done!  Thx!